apps/site/cloudflare/image-loader.ts

```diff
@@ -0,0 +1,22 @@
+import type { ImageLoaderProps } from 'next/image';
+
+const normalizeSrc = (src: string) => {
+  return src.startsWith('/') ? src.slice(1) : src;
+};
+
+export default function cloudflareLoader({
+  src,
+  width,
+  quality,
+}: ImageLoaderProps) {
+  if (process.env.NODE_ENV === 'development') {
+    // Serve the original image when using `next dev`
+    return src;
+  }
+  const params = [`width=${width}`];
+  if (quality) {
+    params.push(`quality=${quality}`);
+  }
+  const paramsString = params.join(',');
+  return `/cdn-cgi/image/${paramsString}/${normalizeSrc(src)}`;
+}
```

Comment:
A few notes with this implementation:
1. is `normalizeSrc` needed? Won't the leading `/` be normalized by the browser? If not, shouldn't there not be a slash / always be a slash passed here? This should _only_ load images, inconsistencies should be resolved at their root, not here?

2. I think this can be simplified to:


```suggestion
import type { ImageLoaderProps } from 'next/image';

// This can also be a `.replace(...)`?
const normalizeSrc = (src: string) => src.startsWith('/') ? src : `/${src}`;

// For tree-shaking, do this outside
export default process.env.NODE_ENV === 'development' ?
  ({ src }) => src :
  ({
    src,
    width,
    quality,
  }: ImageLoaderProps) => `/cdn-cgi/image/width=${width}${quality ? `quality=${quality}` : ''}${normalizeSrc(src)}`;
```

Reply:
I'm happy to go with your version, however as I mentioned there might be benefits in keeping this in sync with the open-next documentation? 🤔 

(PS: if you want you can also try to upstream these changes in the open-next docs (cc. @vicb))

Reply:
Imo, the initial implementation is a lot more readable than this (though normalizeSrc may be better as a just a .replace but I'm either or on it)